<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Minijeu</class>
 <widget class="QDialog" name="Minijeu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>371</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
	border: none;
	background-color: transparent;
	background: transparent;
	padding: 0;
	margin: 0;
	color: #fff;
}

#Minijeu
{
	background-color: #1f232a;
}

#ScoreLabel
{
	margin-right: 3px;
}

QProgressBar
{
	background-color: #2c313c;
	border-style: solid;
	border-radius: 10px;
}

QProgressBar::chunk
{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(246, 137, 41, 255), stop:1 rgba(255, 71, 15, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="HeaderContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignLeft|Qt::AlignTop">
       <widget class="QWidget" name="ScoreContainer" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="ScoreLabel">
           <property name="font">
            <font>
             <family>Modern No. 20</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Score:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ScoreValue">
           <property name="font">
            <font>
             <family>Modern No. 20</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ScoreImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Modern No. 20</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ðŸª™</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignRight|Qt::AlignTop">
       <widget class="QWidget" name="TopCenterContainer" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignRight">
          <widget class="QLabel" name="RulesLabel">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Modern No. 20</family>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Cliquez sur les piÃ¨ces qui apparaitront le plus vite possible !</string>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="StartTimeLabel">
           <property name="font">
            <font>
             <family>Modern No. 20</family>
             <pointsize>20</pointsize>
            </font>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QProgressBar" name="TimeProgressBar">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>0</height>
            </size>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="JeuContainer"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
